HLTrigger/Muon/src/HLTMuonDimuonL3Filter.cc

```diff
@@ -403,20 +322,12 @@ HLTMuonDimuonL3Filter::hltFilter(edm::Event& iEvent, const edm::EventSetup& iSet
 	      for (auto & i : vref) {
 		RecoChargedCandidateRef candref =  RecoChargedCandidateRef(i);
 		TrackRef tktmp = candref->get<TrackRef>();
-		if (tktmp==tk1) {
-		  i1done = true;
-		} else if (tktmp==tk2) {
-		  i2done = true;
-		}
+		if (tktmp==tk1) i1done = true;
+		else if (tktmp==tk2) i2done = true;	//why is this an elif?
```

Comment:
Hi David, it was more of a comment to myself - I was unsure when editing the code why it was written this way in the [old code](https://github.com/cms-sw/cmssw/blob/master/HLTrigger/Muon/src/HLTMuonDimuonL3Filter.cc#L408); I couldn't see how tktmp==tk1 and tktmp==tk2 could be true at the same time (therefore wouldnt need the else if). However as it is harmless, I left it that way